dss/operations/checkout.py

```diff
@@ -0,0 +1,68 @@
+"""
+Tools for managing checkout buckets
+"""
+import typing
+import argparse
+import logging
+
+from dss import Replica
+from dss.operations import dispatch
+from dss import Config
+from cloud_blobstore import BlobNotFoundError
+from dss.api.bundles import get_bundle_manifest
+from dss.storage.hcablobstore import compose_blob_key
+
+
+logger = logging.getLogger(__name__)
+
+checkout = dispatch.target("checkout", help=__doc__)
+
+
+def verify_delete(handler, bucket, key):
+    logger.warning(f'attempting removal of key: {key}')
+    handler.delete(bucket, key)
+    try:
+        handler.get(bucket=bucket, key=key)
+    except BlobNotFoundError:
+        logger.warning(f'Success! unable to locate key {key} in bucket: {bucket} ')
+
+
+@checkout.action("remove_checkout",
+                 arguments={"--replica": dict(choices=[r.name for r in Replica], required=True),
+                            "--keys": dict(nargs="+", help="keys from checkout bucket to remove", required=False),
+                            "--bundle-fqid": dict(nargs="+", help="fqid checkout bucket to remove", required=False)})
+def remove_checkout(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Remove a bundle from the checkout bucket
+    """
+    replica = Replica[args.replica]
+    handler = Config.get_blobstore_handle(replica)
+    bucket = replica.checkout_bucket
+    if args.keys:
+        for _key in args.keys:
+            if 'bundles/' in _key:
+                # get-bundle presignedurl uses key based prefix
+                print(handler.list(bucket, _key))
+                for key in handler.list(bucket, _key):
+                    try:
+                        verify_delete(handler, bucket, key)
+                    except BlobNotFoundError:
```

Comment:
Nit: Would it be less verbose to put the `try: except BlobNotFoundError:` inside of the `verify_delete(handler, bucket, key)` function?